nyc_taxi/nyc_taxi-nongeo.ipynb

```diff
@@ -93,9 +89,8 @@
    "outputs": [],
    "source": [
     "samples = df.sample(frac=1e-4)\n",
-    "scatter = hv.Scatter(samples, 'trip_distance', 'fare_amount')\n",
-    "labelled = scatter.redim.label(trip_distance=\"Distance, miles\", fare_amount=\"Fare, $\")\n",
-    "labelled.redim.range(trip_distance=(0, 20), fare_amount=(0,40)).opts(size=5)"
+    "samples.hvplot.scatter('trip_distance', 'fare_amount', xlabel='Distance, miles', ylabel='Fare, $',\n",
```

Comment:
Could you wrap this line and others so they are a little less wide? So readers don't have to scroll too much when they're on their phone.